CMakeLists.txt

```diff
@@ -100,3 +100,13 @@ endif()
 # *****************************************************************************
 add_subdirectory(src/protobuf)
 add_subdirectory(src)
+
+# *****************************************************************************
+# Copy config files
```

Comment:
We don't need to move everything to the build folder, we just have to move the binary to the root folder, if you mix that you can cause problems if people remove the build folder.

Reply:
I thought it would be easier to copy the config files automatically to the build directory when compiling the code instead of doing otherwise 😅

Reply:
Yep, for some users maybe will be easier, but the build folder should be temporary, and if people start to using it, it will not be safe, as some of the things that we usually say to users is to remove the build folder and start again the build as something on the build folder, some cache for example, can causes some build problems, specially for users that are not very familiar with programming stuff

Reply:
In this case what gonna happen is, every time the user builds the server, the config files will be copied again to the build dir, why would this be a problem? I think it's easier than copying the server binary to the directory where the config files are. This is only for Developer Experience. Have you tried that?